shared-helpers/src/locales/es.json

```diff
@@ -11,7 +11,7 @@
   "account.haveAnAccount": "¿Ya tiene una cuenta?",
   "account.myApplicationsSubtitle": "Vea las fechas de la lotería y los listados de las propiedades para las que ha presentado solicitudes",
   "account.myApplications": "Mis Solicitudes",
-  "account.noApplications": "Parece que todavía no ha realizado una solicitud a ningún anuncio.",
+  "account.noApplications": "Parece que todavía no ha realizado una solicitud a ningún listado.",
   "account.settings.alerts.currentPassword": "Contraseña inválida. Por favor_ vuelva a intentarlo.",
```

Comment:
Spanish translations sometimes use `anuncio` but in discussions w/Ines we decided that `listado` is more appropriate in this context, so I switched it out throughout the translations.